examples/Polynomial_Optimization.ipynb

```diff
@@ -120,24 +145,46 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 3,
    "metadata": {},
-   "outputs": [],
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "termination_status(m) = LOCALLY_SOLVED::TerminationStatusCode = 4\n",
+      "value(a) = 0.00011889415775001936\n",
+      "value(b) = 1.0256125965753116\n",
+      "objective_value(m) = 3.361333003660522e-12\n"
+     ]
+    },
+    {
+     "data": {
+      "text/plain": [
+       "3.361333003660522e-12"
+      ]
+     },
+     "execution_count": 3,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
    "source": [
     "using JuMP\n",
     "using Ipopt\n",
-    "m = Model(solver=IpoptSolver(print_level=0))\n",
-    "@variable m a >= 0\n",
-    "@variable m b >= 0\n",
-    "@constraint m a + b >= 1\n",
+    "m = Model(with_optimizer(Ipopt.Optimizer, print_level=0))\n",
+    "@variable(m, a >= 0)\n",
+    "@variable(m, b >= 0)\n",
+    "@constraint(m, a + b >= 1)\n",
     "peval(a, b) = p(x=>a, y=>a)\n",
```

Comment:
There is a typo here, it is `y=>b`, not `y=>a`

Reply:
This might be the reason we are getting a different solution than above :thinking:

Reply:
I think that this is a typo too, but I am a little confused. As far as I understand, we want to get a different solution than above, i.e. (1,0) or (0,1) at an objective value 0. `y=>a` gives us that and changing it to `y=>b` brings us back to the previous solution, i.e. (0.5, 0.5) with objective value 0.25.  y=>b should be the correct inequality though.

Reply:
We want to show that Sum-of-Squares find the correct optimal solution which NLP finds a suboptimal one. 
The two NLP are two different ways to write an equivalent problem so it makes more sense that it find the same solution than a different one

Reply:
The notebook states that since only first order derivatives are available with registered functions, this modified model finds the correct optimal solution. Should we delete the second model if this is not the case?

Reply:
We should remove this comment about first-order vs second-order derivative. No need to explain why there is a difference since there is not anymore any difference :)

Reply:
Ok, I will update it. Thank you :)